packages/server/modules/core/graph/resolvers/embedTokens.ts

```diff
@@ -0,0 +1,75 @@
+import { db } from '@/db/knex'
+import { Resolvers } from '@/modules/core/graph/generated/graphql'
+import {
+  listProjectEmbedTokensFactory,
+  revokeEmbedTokenByIdFactory,
+  storeApiTokenFactory,
+  storeEmbedApiTokenFactory,
+  storeTokenResourceAccessDefinitionsFactory,
+  storeTokenScopesFactory
+} from '@/modules/core/repositories/tokens'
+import {
+  createEmbedTokenFactory,
+  createTokenFactory
+} from '@/modules/core/services/tokens'
+import { mapAuthToServerError } from '@/modules/shared/helpers/errorHelper'
+import { removeNullOrUndefinedKeys } from '@speckle/shared'
+
+export = {
+  Project: {
+    embedTokens: async (parent, _args, context) => {
+      const canReadEmbedTokens =
```

Comment:
all of these should check resourceAccessRules to ensure the user even is allowed to work w/ embed tokens for this particular project

Reply:
This one I don't follow, I only see resource access support for workspace and projects. Is this a check that the token allows access to the project, if resource limit exist?